be/src/exec/vectorized/hdfs_scanner.cpp

```diff
@@ -217,7 +217,6 @@ void HdfsScannerContext::update_not_existed_columns_to_chunk(vectorized::ChunkPt
     if (not_existed_slots.size() == 0) return;
 
     ChunkPtr& ck = (*chunk);
-    ck->set_num_rows(row_count);
```

Comment:
There is still a small bug. When a new value is inserted into an existing data file partition after adding a column, the partition column is 0.